TYPO3-Solr/ext-solr
# [TASK] Add extension configuration to documentation

# What this pr does

Adds a brief description and a screenshot of the extension
configuration to the extension documentation.

# How to test

Check the documentation section  Getting Started =>  Configure Extension.

Resolves: #2893
